frontend/src/app/supportAdmin/DeviceType/DeviceForm.tsx

```diff
@@ -117,93 +117,95 @@ const DeviceForm = (props: Props) => {
 
   const createSupportedDiseaseRow = (index: number) => {
     return (
-      <div className="grid-row grid-gap">
-        <div className="tablet:grid-col">
-          <Select
-            label={"Supported disease"}
-            options={props.supportedDiseaseOptions}
-            value={
-              device?.supportedDiseaseTestPerformed?.[index]
-                ?.supportedDisease || ""
-            }
-            onChange={(val) => {
-              if (device?.supportedDiseaseTestPerformed) {
-                let newSupportedDisease = [
-                  ...device?.supportedDiseaseTestPerformed,
-                ];
-                newSupportedDisease[index].supportedDisease = val;
-                updateDevice({
-                  ...device,
-                  supportedDiseaseTestPerformed: newSupportedDisease,
-                });
+      <fieldset className={"usa-fieldset"}>
```

Comment:
I think we need to add a `<legend>` so we will see a change in the visual 🤔 https://cdc-devtools.dequecloud.com/rule-help/en/label-group-not-present (let me know if you can see this link 👀)